google/stackdriver_monitoring/metric_collector_handlers.py

```diff
@@ -1,24 +1,47 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from datetime import datetime
-import cgi
 import json
-import logging
-from stackdriver_client import StackdriverClient
+from datetime import datetime
+
+
+def accepts_html(request):
+  if not hasattr(request, 'headers'):
+    return False
+  accept = request.headers.get('accept', None)
+  if not accept:
+    return None
+  return 'text/html' in accept.split(',')
```

Comment:
This probably isn't sufficient if you look at all the ways your header can be formatted: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html